### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Kénan Augsburger, Mário Ferreira

### DISCUSSION THREAD

**Discussion by Mondotosz** [heig-vd-dai-course/heig-vd-dai-course#483] [Show and tell]
Our project is to create an "FTP" server and client.

The functionalities would be the following:

- List the files/folders on the server
- Get a file from the server
- Store a file on the server
- Delete a file on the server

If there is enough time left we would also try to secure the server with a password to prevent unauthorized access

We will try to use git-flow and automatically push the docker images using github actions

- link to the repo [Thynkon/dai-pw-02](https://github.com/Thynkon/dai-pw-02)
- link to the release commit [e03bf77](https://github.com/Thynkon/dai-pw-02/commit/e03bf771f8e503e7851b95e2462b269205bac975)

**Comment by ludelafo** [comment]
Looks good for us!
